boards/makerbase/mks_canable_v20/doc/index.rst

```diff
@@ -0,0 +1,173 @@
+.. zephyr:board:: mks_canable_v20
+
+Overview
+********
+
+The Makerbase MKS CANable V2.0 board features an ARM Cortex-M4 based STM32G431C8 MCU
+with a wide range of connectivity support and configurations.
```

Comment:
A range of of? Not on this board, so I think that can be removed from the board description.

Reply:
MCU supports `a range of...`. But in terms of board I agree and will update